README.md

```diff
@@ -11,7 +11,28 @@
 <h5>pip3 install threader3000</h5>
 <h5>Run by typing "threader3000"</h5>
 <h4>Install via Git</h4>
-<h5>git clone https://github.com/dievus/threader3000.git to save the program to your machine, or utilize the download option.</h5>
```

Comment:
Changes to the README are already going to be a part of the next push, and no changes will be made at this time per this request.  I do appreciate the input and suggestions.